dist2src.py

```diff
@@ -13,10 +13,16 @@
 import sh
 from packit.config.package_config import get_local_specfile_path
 from rebasehelper.specfile import SpecFile
-
+from yaml import dump
 
 logger = logging.getLogger(__name__)
 
+# directory with downstream files
+DOWNSTREAM_FILES_DIR = "centos-packaging"
+# build/test
+TARGETS = ["centos-stream-x86_64"]
+SG_START_TAG = "sg-start"
```

Comment:
Maybe spell this out, I find abbreviation in this case unnecessary and making the code more difficult to understand. (read "sg" like [Star Gate](https://en.wikipedia.org/wiki/Stargate) :smile:)

Reply:
it's `START_TAG` now